src/services/infra/InfraService.ts

```diff
@@ -387,7 +404,9 @@ export default class InfraService {
   }
 
   siteUpdate = async () => {
-    const messages = await this.queueService.pollMessages()
+    const messages = DEPRECATE_SITE_QUEUES
+      ? await this.dynamoDBService.getAllCompletedLaunches()
```

Comment:
if any of this throws, are the errors caught appropriately?

Reply:
using a trivial try-catch wrapper here. Understand that we have an error handling portion to standardise the error formatting, will modify that when the scaffolding work for the error formatting is up! 0fa79124d71a6e8ec04bc0c698aff6b18f0260dd